apps/aechannel/test/aesc_txs_SUITE.erl

```diff
@@ -442,59 +442,57 @@ create_same_account(_Cfg) ->
     %% Test channel with oneself is not allowed
     TxSpecI = aesc_test_utils:create_tx_spec(
                 PubKey, PubKey,
-                #{initiator_amount => 100,
-                  channel_reserve => 2,
-                  fee => 20000}, S),
+                #{}, S),
     {ok, TxI} = aesc_create_tx:new(TxSpecI),
     {error, initiator_is_responder} = aetx:process(TxI, Trees, Env),
     ok.
 
 create_insufficient_funds(_Cfg) ->
-    {Loaded, NotLoaded, S} = create_loaded_accounts(100, 1),
+    {Loaded, NotLoaded, S} = create_loaded_accounts(60000 * aec_test_utils:min_gas_price(),
+                                                    1),
     Trees = aesc_test_utils:trees(S),
     Height = 1,
     Env = aetx_env:tx_env(Height),
 
     %% Test insufficient initiator funds
     TxSpecI = aesc_test_utils:create_tx_spec(
                 NotLoaded, Loaded,
-                #{initiator_amount => 1,
-                  fee => 50000}, S),
+                #{initiator_amount => 2,
+                  fee => 50000 * aec_test_utils:min_gas_price()}, S),
     {ok, TxI} = aesc_create_tx:new(TxSpecI),
     {error, insufficient_funds} = aetx:process(TxI, Trees, Env),
 
     %% Test insufficient responder funds
     TxSpecR = aesc_test_utils:create_tx_spec(
                 Loaded, NotLoaded,
-                #{initiator_amount => 1,
-                  fee => 50000}, S),
+                #{responder_amount => 2,
+                  fee => 50000 * aec_test_utils:min_gas_price()}, S),
     {ok, TxR} = aesc_create_tx:new(TxSpecR),
     {error, insufficient_funds} = aetx:process(TxR, Trees, Env),
 
     ok.
 
 
 create_insufficient_funds_reserve(_Cfg) ->
-    {Loaded, NotLoaded, S} = create_loaded_accounts(100000, 100000),
+    {Loaded1, Loaded2, S} = create_loaded_accounts(100000 * aec_test_utils:min_gas_price(),
+                                                   100000 * aec_test_utils:min_gas_price()),
     Trees = aesc_test_utils:trees(S),
     Height = 1,
     Env = aetx_env:tx_env(Height),
 
     %% Test initiator funds lower than channel reserve
     TxSpecI = aesc_test_utils:create_tx_spec(
-                NotLoaded, Loaded,
+                Loaded1, Loaded2,
                 #{initiator_amount => 1,
-                  channel_reserve => 2,
-                  fee => 50000}, S),
+                  channel_reserve => 2}, S),
     {ok, TxI} = aesc_create_tx:new(TxSpecI),
     {error, insufficient_initiator_amount} = aetx:process(TxI, Trees, Env),
 
     %% Test responder funds lower than channel reserve
     TxSpecR = aesc_test_utils:create_tx_spec(
-                Loaded, NotLoaded,
+                Loaded1, Loaded2,
                 #{responder_amount => 1,
-                  channel_reserve => 2,
-                  fee => 50000}, S),
+                  channel_reserve => 2}, S),
```

Comment:
Same here, if the channel reserve is lower than the minimum fee, this in itself might throw an error. After all, the channel reserve is partly there, I believe, to pay for a transaction

Reply:
The only transaction that consumes the channel's balance for fee is the close mutual one. Since this is the channel create tx, the fee is being payed by the `initiator` only so the check is being done that both the `initiator`/`responder`'s balances are greater than the channel reserve. I think we test the correct thingy, yes